wormhole-connect/src/views/v2/Bridge/AssetPicker/SearchableList/SearchInput.tsx

```diff
@@ -34,42 +35,44 @@ export default function SearchInput(props: SearchInputProps) {
   const { classes } = useStyles();
 
   return (
-    <TextField
-      className={classes.input}
-      autoFocus
-      fullWidth
-      inputProps={{
-        style: {
-          fontSize: 16,
-          height: 22,
-          lineHeight: 22,
-        },
-      }}
-      placeholder={props.placeholder}
-      size="small"
-      sx={{
-        // Root class for the input field
-        '& .MuiOutlinedInput-root': {
-          // Class for the input placeholder text
-          '& .MuiOutlinedInput-input::placeholder': {
-            fontWeight: 300,
+    <Box sx={{ padding: '0 16px' }}>
```

Comment:
per design, the asset list items are supposed to go edge-to-edge. so I moved some padding properties around here.